astrology.lic

```diff
@@ -276,15 +285,30 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
-    check_astral
-    check_weather
-    check_events(check_pools) if @predict_event
-    check_moons
-    check_heavens
+  def train_astrology(settings)
+    return unless settings
+    return unless settings.astrology_training
+
```

Comment:
I think it makes more sense to check if its `nil?` and not `[]`. This means they have updated astrology via dependency but not reloaded base.yaml